### DISCUSSION TITLE
Trouble with git clone via SSH

### DISCUSSION THREAD

**Discussion by FrancoisFr** [community/community#22372] [New to GitHub]
<p>Hello, so, i am trying to git clone a repo on a new machine. i created an ssh key, added it to the ssh-agent, added the .pub key to my settings in github.</p>
<p>When i do ssh <a href="mailto:git@github.com">git@github.com</a> it works</p>
<p>When i try to git clone one of my repo using ssh, it does not.</p>
<p>Does anyone has an idea as to why the heck it is not working ?</p>

**Comment by airtower-luna** [comment]
<p>Please provide logs, otherwise it’s hard to even guess what might be going wrong.</p>

**Comment by FrancoisFr** [comment]
<p>this is what i get when i try to clone :</p>
<p><a href="mailto:git@github.com">git@github.com</a>: Permission denied (publickey).<br>
fatal: Could not read from remote repository.</p>
<p>Please make sure you have the correct access rights<br>
and the repository exists.</p>
<p>And this is what i get when i try to ssh into github, without cloning my repo :</p>
<p>PTY allocation request failed on channel 0<br>
Hi FrancoisFr! You’ve successfully authenticated, but GitHub does not provide shell access.<br>
Connection to <a href="http://github.com" rel="noopener nofollow ugc">github.com</a> closed</p>
<p>So, as i said, i can connect in ssh on github, but it won’t work when i try to clone my own freaking repo x)</p>

**Comment by FrancoisFr** [comment, marked as answer]
<p>found a fix doing this :  <a href="https://github.community/t/git-via-ssh-from-windows-returns-permission-denied/1107/11" class="inline-onebox">Git via SSH from Windows returns Permission Denied - #11 by lzag</a></p>

**Reply by Halllker** [reply]
Any chance you remember what was the cause? I have the same issue but the link you've provided leads nowhere now.
